Sentry/src/test/java/io/sentry/util/FileUtilsTest.kt

```diff
@@ -0,0 +1,58 @@
+package io.sentry.util
+
+import org.junit.Test
+import java.io.File
+import java.nio.file.Files
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNull
+
+class FileUtilsTest {
+
+    @Test
+    fun `deleteRecursively returns true on non-existing file or null`() {
+        assert(FileUtils.deleteRecursively(null))
+        assert(FileUtils.deleteRecursively(File("")))
+    }
+
+    @Test
+    fun `deleteRecursively deletes a simple file`() {
+        val f = Files.createTempFile("here", "test").toFile()
+        assert(f.exists())
+        assert(FileUtils.deleteRecursively(f))
+        assertFalse(f.exists())
+    }
+
+    @Test
+    fun `deleteRecursively deletes a folder`() {
+        val d = Files.createTempDirectory("here").toFile()
+        val f = File(d, "test")
+        val d2 = File(d, "dir2")
+        val f2 = File(d2, "test")
```

Comment:
I'd give proper names to the variables, and jump a line to show the intention of your tests.

```
// setup vars

// call methods

// assert values
```